registry.json

```diff
@@ -1741,5 +1741,30 @@
       "clientPublic": "https://rpc.ftm.tools",
       "clientDocs": "https://eth.wiki/json-rpc/API"
     }
+  },
+  {
+    "id": "cryptoorg",
+    "name": "Crypto.org",
+    "coinId": 394,
+    "symbol": "CRO",
+    "decimals": 8,
+    "blockchain": "Cosmos",
+    "derivationPath": "m/44'/394'/0'/0/0",
```

Comment:
can you cross check with keplr extension wallet? should be compatible

Reply:
Yes, confirmed, Kepl Chrome extension wallet results in the same addresses.